source/configuration_manual/mail_location/imapc/index.rst

```diff
@@ -49,22 +49,31 @@ Connection Settings
    A space-separated list of features, optimizations, and workarounds that can
    be enabled.
 
+   .. versionchanged:: v2.4.0;v3.0.0 Several features are now automatically
+                       enabled and the respective flags dropped. In their place
+                       new flags to disable these features were added.
+
    **Features**
 
-   ``acl``
+   ``no-acl``
+
+     If the imap-acl plugin is loaded, the imapc acl feature is automatically
+     enabled. With it IMAP ACL commands (MYRIGHTS, GETACL, SETACL, DELETEACL)
+     are proxied to the imapc remote location.
```

Comment:
Could also continue (this is arguably a bug): Note that currently these commands are attempted to be used even if the remote IMAP server doesn't advertise the ACL capability.